app/packages/app/package.json

```diff
@@ -18,6 +18,7 @@
     "@emotion/styled": "^11.10.5",
     "@fontsource/roboto": "^4.5.8",
     "@kobsio/core": "*",
+    "@kobsio/harbor": "*",
```

Comment:
Please run a `npm run init` and commit the lock file. my package-lock.json changed after running an install.